Dockerfile

```diff
@@ -19,22 +19,12 @@ ENV GO111MODULE=on \
     GOARM=${TARGETVARIANT}
 
 WORKDIR /go/src/github.com/open-policy-agent/gatekeeper
-
-COPY pkg/ pkg/
-COPY third_party/ third_party/
-COPY vendor/ vendor/
-COPY main.go main.go
-COPY apis/ apis/
-COPY go.mod .
-
-RUN go build -mod vendor -a -ldflags "${LDFLAGS:--X github.com/open-policy-agent/gatekeeper/pkg/version.Version=latest}" -o manager main.go
+COPY . .
```

Comment:
This runs the risk of copying temp files, such as `.output` into the system, which may bloat the binary and/or lead to inadvertent disclosures of a publisher's machine.

Reply:
this is just the builder image, it is not published

Reply:
ack, makes sense